src/main/java/javaClasses/homework_5/Heghine_Khachatryan/interface_part/Railway.java

```diff
@@ -0,0 +1,68 @@
+package javaClasses.homework_5.Heghine_Khachatryan.interface_part;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Railway implements Bookable, Stoppable {
+    private String transportName;
+    private String name;
+    private String surname;
+    private String passport;
+
+    public Railway(String transportName, String name, String surname, String passport) {
+        this.transportName = transportName;
+        this.name = name;
+        this.surname = surname;
+        this.passport = passport;
+    }
+
+    @Override
+    public String bookTicket() {
+        String regex = "[A-Z]{2}[0-9]{7}";
+        Pattern pattern = Pattern.compile(regex);
+        Matcher matcher = pattern.matcher(getPassport());
+        if (!matcher.matches()) {
+            return "Passport number doesn't match. Please check again";
```

Comment:
Excellent job on using regular expression! Actually if a part of code contains checking for parameters, you can actually throw a Runtime exception if that checking is failed. So that will indicate failure through process. Overall, nice job!